app/scripts/lib/marketing-email-client.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A client to talk to the basket marketing email server
+ */
+
+'use strict';
+
+define([
+  'lib/xhr',
+  'lib/promise',
+  'lib/constants'
+], function (xhr, p, Constants) {
+
+  var NEWSLETTER_ID = Constants.MARKETING_NEWSLETTER_ID;
+
+  function MarketingEmailClient(options) {
+    options = options || {};
+
+    var self = this;
+
+    self._xhr = options.xhr || xhr;
+    self._baseUrl = options.baseUrl;
+  }
+
+  MarketingEmailClient.prototype = {
+    _request: function (path, type, accessToken, data, headers) {
```

Comment:
Looks like this takes an existing accessToken.  What scopes does this token have?  ISTM we may want to use a special-purpose scope for this, e.g. `basket` instead of the usual `profile` scope.